src/components/DisplayBooks.module.css

```diff
@@ -0,0 +1,128 @@
+.root {
+  --icon-font: montserrat;
+  --font: robotoslab;
+}
+
+.booksContainer {
+  list-style-type: none;
+  padding: 0;
+}
+
+.liCard {
+  height: 50%;
+  padding: 1rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 0.8rem;
+}
+
+.bookCard {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.title {
+  font-family: var(--font);
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0;
+}
+
+.category {
+  font-family: var(--font);
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  margin: 0;
+}
+
+.buttons {
+  font-family: var(--font);
+  font-size: 0.875rem;
+  color: #4386bf;
+  border: none;
+  background-color: white;
+  margin-top: 1rem;
+  padding: 0;
+}
+
+.completedDiv {
+  display: flex;
+  align-items: center;
+}
+
+.circle {
+  width: 2.5rem;
+  height: 2.5rem;
+  border: 3px solid #307bbe;
+  border-radius: 50%;
+  margin-right: 1rem;
+}
+
+.percentage {
+  font-family: var(--icon-font);
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin: 0;
+}
+
+.completed {
+  opacity: 0.5;
+  font-family: var(--icon-font);
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin: 0;
+}
+
+.current {
+  opacity: 0.5;
+  font-family: var(--font);
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter {
+  font-family: var(--font);
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
```

Comment:
- [ ] Kindly combine these two selectors because they're sharing almost the same rules, the DRY rule is about don't repeat yourself, and this is for cleaner and more optimized code 😄 🙏

_You can find about best practices right [here](https://speckyboy.com/good-bad-css-practices/)._

>Please apply these changes to all similar situations through your code files!